nil/services/nilservice/service.go

```diff
@@ -12,7 +12,6 @@ import (
 	"time"
```

Comment:
This patch (RawAPI has been redesigned. Added the ability to operate any number of different ShardAPIs in any combination) also needs a good description.

Reply:
Writing commit descriptions is what gave me the most trouble yesterday after rebase) I'll try to make the descriptions more detailed a bit later.